release_notes/40-release-notes.adoc

```diff
@@ -0,0 +1,476 @@
+- Vulnerability scanning support for non-RPM (ASP).NET Core Runtime is deprecated and is planned for removal in the {product-title-short} 4.2.0 release.
+- All `/v1/report` APIs for creating and managing vulnerability reports are deprecated and will be replaced with new `/v2/report` APIs in a future release.
+
+[id="remove-kernel-rn40"]
+==== Remove kernel module as collection method
+
+Currently, secured clusters can specify three options of collection methods for runtime events: eBPF (selected by default), kernel module, or no collection. Kernel module as a collection method was deprecated in the {product-title-short} version 3.74 release and is planned for removal in the {product-title-short} version 4.1 release.
+
+[discrete]
+[id="actions-kernel-module-rn40"]
+===== Actions you must take
+Verify the collection method of your secured clusters. This value is set in the `collector.collectionMethod` parameter and is one of the following methods:
+
+- `EBPF`
+- `KERNEL_MODULE`
+- `NO_COLLECTION`
+
+If any of your secured clusters uses `KERNEL_MODULE` as a collection method, change it to `EBPF`.
+
+[id="removed-features-rn40"]
+== Removed features
+
+The following section provides additional information about removed features listed in the preceding table.
+
+[id="permissions-rn40"]
+=== Permissions and permission sets
+
+As announced in the Release Notes for {product-title-short} 3.73.0, some permissions are now grouped for simplification. The deprecation process in 4.0 will remove and replace the deprecated permissions with the replacing permission. The access level granted to the replacing permission will be the lowest among all access levels of the replaced permissions.
+
+The following list provides more information about permission and permission set changes:
+
+* Permission `Administration` replaces the deprecated following permissions:
+** `AllComments`
+** `Config`
+** `DebugLogs`
+** `NetworkGraphConfig`
+** `ProbeUpload`
+** `ScannerBundle`
+** `ScannerDefinitions`
+** `SensorUpgradeConfig`
+** `ServiceIdentity`
+* Permission `Compliance` replaces the deprecated permission `ComplianceRuns`.
+* The `Analyst` permission set will change behavior. Instead of allowing `read` access to all resources except `DebugLogs`, it will allow `read` access to all resources except `Administration`. This change affects you if you were using the `Analyst` role or permission set for actions requiring `read` access on the following resources:
++
+--
+** `AllComments`
+** `Config`
+** `NetworkGraphConfig`
+** `ProbeUpload`
+** `ScannerBundle`
+** `ScannerDefinitions`
+** `SensorUpgradeConfig`
+** `ServiceIdentity`
+--
++
+To address this change, preemptively create a new permission set with `read` access on the `Administration` and other required resources, and reference it instead of `Analyst` in the created roles.
+
+[id="bug-fixes-rn40"]
+== Bug fixes
+
+[id="resolved-in-version-40"]
+=== Resolved in version 4.0
+
+Release date: 1 May 2023
```

Comment:
Above, it lists a [May 8 release date](https://github.com/openshift/openshift-docs/pull/59283/files#diff-ed71c5b403507773c9c5babf3339d353bc04209c35dddc6c52258a2638a4d8b0R15). Does that conflict with this date?

Reply:
Yes, I should have changed it there - it's not May 1 now, but we're not sure when it will actually be. TY for catching this!

Reply:
I actually am going to un-label this for merging since we don't have a date for the release now, and send it for merge review when I know what date to use.